boutdata/data.py

```diff
@@ -487,6 +487,47 @@ def format_inline_comment(name, options):
 
         return f.getvalue()
 
+    def get_bool(self, name, default=None):
+        """
+        Convert an option value to a bool, in (almost) the same way as BOUT++.
+
+        Note BOUT++ will convert any option value beginning with "y", "Y", "t", "T" or
```

Comment:
Would it make sense to put this under a 

```
    Warnings
    ----------
```

heading. I think this will render it as an admonition inside a highlighted box